Uchu.World/Client/CdClient/ClientCache.cs

```diff
@@ -51,6 +51,10 @@ public static class ClientCache
         /// </summary>
         public static async Task LoadAsync()
         {
+            // Update the CDClient database.
+            await using var cdContext = new CdClientContext();
+            await cdContext.EnsureUpdatedAsync();
```

Comment:
I haven't tested the effect on load times with this change. We could do a code-level change instead of a data-level change, but I am not sure about having hard-coded changes like that.

Reply:
I don't think we should do a hardcoded change no. But (as I understand it) at [WorldUchuServer.cs:87](https://github.com/UchuServer/Uchu/blob/caa7be2799eacba313f0c30835e50adb3d5ddf05/Uchu.World/WorldUchuServer.cs#L87) the table cache is loaded on a separate thread from the one that loads the zone objects, so I was wondering if this could be a problem if it's trying to use the ClientCache before the migrations have completed.

Reply:
Fair point. There actually would be a problem if migration was required to the Zone table since the first read is always done before the CdClient cache is created. I've moved it earlier in the process so that won't happen.